app/DoctrineMigrations/Version20160313161454.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20160313161454 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE fos_user CHANGE want_to_learn want_to_learn TINYINT(1) DEFAULT NULL');
```

Comment:
@jongotlin You know what is really wrong with the $wantToLearn variable? It's how you assign it. Do you really think that it should be a boolean value? 

'user.form.want_to_learn.choice.learn'  => true,
 'user.form.want_to_learn.choice.teach'  => false,